index.js

```diff
@@ -91,13 +101,15 @@ function checkAndReport(node, timeout) {
 					}
 					console.groupCollapsed('%c%s: %c%s %s', fmt, result.impact, defaultReset, result.help, result.helpUrl);
 					result.nodes.forEach(function (node) {
-						var el = document.querySelector(node.target.toString());
-						console.error(node.failureSummary);
-						console.log('HTML: %c%s', boldCourier, node.html);
-						if (!el) {
-							console.log('Selector: %c%s', boldCourier, node.target.toString());
-						} else {
-							console.log('Element: %o', el);
+						console.error(node.any[0].message);
```

Comment:
This is not sufficient code to generate the summary correctly. Check out this code - which you may have to duplicate if it is not exposed via the `axe.commons` or `axe.utils` objects

https://github.com/dequelabs/axe-core/blob/master/lib/core/reporters/helpers/failure-summary.js